Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM debian:stretch-20180213
+
+# Install all dependencies from manifest.scm
+RUN apt-get update && apt-get install -y \
+    autoconf \
+    automake \
+    bash \
+    bison \
+    cmake \
+    coreutils \
+    curl \
+    device-tree-compiler \
+    diffutils \
+    doxygen \
+    expat \
+    file \
+    findutils \
+    flex \
+    gawk \
+    gcc-6 \
+    g++ \
+    git \
+    libgmp-dev \
+    make \
+    grep \
+    gzip \
+    libedit-dev \
+    libssh2-1-dev \
+    libtool \
+    libusb-1.0-0-dev \
+    libxml2-dev \
+    mpc \
+    libmpfr-dev \
+    ninja-build \
+    libncurses-dev \
+    libssl-dev \
+    patchelf \
+    pkg-config \
+    python2.7 \
+    sed \
+    swig \
+    tar \
+    texinfo \
+    zlib1g-dev \
+    zsh
+
+RUN ln -s /usr/bin/python2.7 /usr/bin/python2
+RUN git clone https://github.com/riscv-rust/riscv-rust-toolchain.git
```

Comment:
Note this clones it into `/`. Perhaps it should clone it into `/root` (or perhaps even a non-root user!)